Cargo.toml

```diff
@@ -172,6 +173,7 @@ regex = { version = "1.11.1", default-features = false }
 regex-syntax = { version = "0.8.4", default-features = false }
 reqwest = { version = "0.12.9", default-features = false }
 reqwest-eventsource = { version = "0.6.0", default-features = false }
+rexpect = "0.6.0"
```

Comment:
Can you create an issue to update the cli tests for device/user invitation dance ?

Reply:
#9145